src/qiskit_sphinx_theme/assets/styles/_drop-shadows.scss

```diff
@@ -15,6 +15,11 @@
 //
 // `:not(#specifity-war)` is to work around specifity wars.
 
```

Comment:
Even though this code only impacts the Ecosystem theme, I put it in the Qiskit theme code. It's not worth explaining the complexity of why we wouldn't need this code in Qiskit and only the Ecosystem. The code fits logically in this file `_drop-shadows.scss`.